### PR TITLE
[release-1.19] Update to use 1.19.6 for testing

### PR DESCRIPTION
**Pull request by ericvn**
Update to use 1.19.6 for testing

### CODE REVIEW

**Comment by ericvn** [comment]
/test doc.test.profile-none

timed out

**Comment by ericvn** [comment]
Failure is in `tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh`. First time the test failed (took 142 secs), the next two had timeouts in the test.

**Comment by ericvn** [comment]
profile-none failure in tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh looking like a timeout:
```
+++__verify_with_retry(): snip_apply_weightbased_tcp_routing_4
++++snip_apply_weightbased_tcp_routing_4(): kubectl get pod -l app=sleep -n istio-io-tcp-traffic-shifting -o 'jsonpath={.items..metadata.name}'
+++snip_apply_weightbased_tcp_routing_4(): export SLEEP=sleep-9454cc476-h2s2h
+++snip_apply_weightbased_tcp_routing_4(): SLEEP=sleep-9454cc476-h2s2h
+++snip_apply_weightbased_tcp_routing_4(): for i in {1..20}
+++snip_apply_weightbased_tcp_routing_4(): kubectl exec sleep-9454cc476-h2s2h -c sleep -n istio-io-tcp-traffic-shifting -- sh -c '(date; sleep 1) | nc 172.18.7.110 31400'
```

**Comment by ericvn** [comment]
I did look into the failure in an earlier PR when moving to 1.20, and the solution there was to move the test as there might be something in the earlier tests causing issues. Possibly, something was cherry-picked back which is causing the failure to occur in 1.19 as well. I can look at trying the same thing in 1.19 that was done in 1.20. so I don't think anyone figured out the underlying issue.

**Comment by ericvn** [comment]
/test doc.test.profile-none

Failing tests are ones that passed earlier.

**Comment by ericvn** [comment]
√

**Comment by ericvn** [comment]
/test doc.test.profile-none

Looking at the artifacts, there seem to be various flakes. No test is consistently failing after moving the tcp-traffic-shifting to a different profile.

**Comment by ericvn** [comment]
/test doc.test.profile-none

Last test failure was `TestDocs/tasks/traffic-management/ingress/gateway-api/test.sh` for third time.

File: content/en/docs/tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@
 source "tests/util/gateway-api.sh"
 install_gateway_api_crds
 
-# @setup profile=none
+# @setup profile=minimal
```
Comment (kfaseela) [inline content/en/docs/tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh:21]: hope this change is intended?

Comment (ericvn) [reply content/en/docs/tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh:21]: It is. See #14091 for the reason it was made in `master` and `1.20`.

**Review by dhawton — APPROVED** [review]
LGTM

**Review by kfaseela — APPROVED** [review]

**Pull request merged**